Dockerfile

```diff
@@ -0,0 +1,35 @@
+FROM ubuntu:22.04
+
+RUN \
+    apt update && \
+    DEBIAN_FRONTEND=noninteractive apt-get install -y \
+        build-essential \
+        verilator \
+        git \
+        apt-transport-https \
+        curl \
+        gnupg 
+
+RUN \
+    apt update && \
+    DEBIAN_FRONTEND=noninteractive apt-get install -y \
+        zip \
```

Comment:
Why zip/unzip?

Reply:
Because `RUN curl -s "https://get.sdkman.io" | bash` will fail with the following error

```
 > [4/7] RUN curl -s "https://get.sdkman.io" | bash:
0.732 Looking for a previous installation of SDKMAN...
0.732 Looking for unzip...
0.733 Not found.
0.733 ======================================================================================================
0.733  Please install unzip on your system using your favourite package manager.
0.733 
0.733  Restart after installing unzip.
0.733 ======================================================================================================
```

Reply:
Add comments.

Reply:
Done as requested